src/main/java/org/mitre/synthea/world/agents/Provider.java

```diff
@@ -214,14 +218,18 @@ public static void loadProviders(String state, String abbreviation, String filen
           || (abbreviation != null && abbreviation.equalsIgnoreCase(currState))) {
         Provider parsed = csvLineToProvider(row);
         
-        
-        parsed.servicesProvided.add(Provider.AMBULATORY);
-        parsed.servicesProvided.add(Provider.INPATIENT);
-        parsed.servicesProvided.add(Provider.WELLNESS);
-        if ("Yes".equals(row.remove("emergency"))) {
-          parsed.servicesProvided.add(Provider.EMERGENCY);
+
+        if (filename.equals("providers/hospitals.csv") || filename.equals("providers/va_facilities.csv")) {
+          parsed.servicesProvided.add(Provider.AMBULATORY);
+          parsed.servicesProvided.add(Provider.INPATIENT);
+          parsed.servicesProvided.add(Provider.WELLNESS);
+          if ("Yes".equals(row.remove("emergency"))) {
+            parsed.servicesProvided.add(Provider.EMERGENCY);
+          }
+        }
```

Comment:
Style: `} else if` code should continue on same line. Surprised this didn't generate a warning.